Tribler/community/chant/chant.py

```diff
@@ -0,0 +1,129 @@
+import os
+from datetime import datetime
+
+from libtorrent import add_files, bdecode, bencode, create_torrent, file_storage, set_piece_hashes
+from Tribler.community.chant.MDPackXDR import serialize_metadata_gossip, \
+    deserialize_metadata_gossip, CHANNEL_TORRENT, MD_DELETE
+
+from Tribler.community.chant.orm import MetadataGossip, Peer, known_sig, known_pk, trusted_pk
+
+channels_dir = "/tmp"
```

Comment:
This should be the Tribler state directory I suppose?

Reply:
Yeah, this is just a placeholder, until I properly integrate the stuff with Tribler.